Tools/_sunrise/changelog/changelog_path.py

```
@@ -0,0 +1,21 @@
+
+    raw_path = value.strip()
+    posix_path = PurePosixPath(raw_path)
+    windows_path = PureWindowsPath(raw_path)
+    path = Path(raw_path)
+    if (
+        posix_path.is_absolute()
+        or windows_path.is_absolute()
+        or ".." in posix_path.parts
+        or ".." in windows_path.parts
+    ):
+        raise RuntimeError("CHANGELOG_FILE должен быть относительным путём внутри репозитория")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)